msal/src/main/java/com/microsoft/identity/client/SignInParameters.java

```diff
@@ -0,0 +1,46 @@
+package com.microsoft.identity.client;
+
+import android.app.Activity;
+
+import androidx.annotation.Nullable;
+
+import lombok.Builder;
+import lombok.Data;
+import lombok.NonNull;
+import lombok.Singular;
+import java.util.List;
+
+/**
+ * Encapsulates the parameters for calling signIn() in SingleAccountPublicClientApplication.
```

Comment:
Nit: can we add a link here to SingleAccountPublicClientApplication and signIn()?